baybe/campaign.py

```diff
@@ -567,15 +558,16 @@ def recommend(
 
         # Get the recommended search space entries
         try:
-            # NOTE: The `recommend` call must happen on `self.recommender` to update
-            #   potential inner states in case of meta recommenders!
-            rec = self.recommender.recommend(
-                batch_size,
-                searchspace,
-                self.objective,
-                self._measurements_exp,
-                None if is_nonpredictive else pending_experiments,
-            )
+            with Settings(preprocess_dataframes=False):
```

Comment:
Note to myself
This currently causes troubles since the random seed logic is still incorrect: even when omitting the `random_seed` argument, the random state is altered because the seed is then extracted from the `active_settings` (or the env vars), triggering a change of random state